src/setup.jl

```diff
@@ -97,14 +87,14 @@ function def_binary_op(tin, tout, jlname, jlname!, mklname, broadcast)
         $(isempty(exports) ? nothing : Expr(:export, exports...))
         function ($jlname!)(out::Array{$tout,N}, A::Array{$tin,N}, B::Array{$tin,N}) where {N}
             size(out) == size(A) == size(B) || $(broadcast ? :(return broadcast!($jlname, out, A, B)) : :(throw(DimensionMismatch())))
```

Comment:
What is this `broadcast` business about? That seems odd to me, it doesn't even call VML? Just remove?

Reply:
More legacy code, maybe something that was a thing before Julia 0.7. I will have a look

Reply:
It appears to have been some construct to maybe allow something like `IVM.atan(rand(10,10), rand(100,1))`
But guessing at the users intention here seems wrong, so instead now properly throwing a dimension mismatch error.